zingolib/src/testutils.rs

```diff
@@ -615,6 +623,56 @@ pub(crate) fn timestamped_test_log(text: &str) {
     println!("{}: {}", crate::utils::now(), text);
 }
 
+#[allow(unused_macros)]
+macro_rules! build_method {
+    ($name:ident, $localtype:ty) => {
+        #[doc = "Set the $name field of the builder."]
+        pub fn $name(&mut self, $name: $localtype) -> &mut Self {
+            self.$name = Some($name);
+            self
+        }
+    };
+}
+
+#[allow(unused_imports)]
+pub(crate) use build_method;
```

Comment:
maybe this could be moved down so we have 3macro-3export instead of 1macro-1export-2macro-2export

Reply:
i assume it was like this already because the first was used in production code but no longer is used

Reply:
Fixed.